tests/fixers/test_use_l10n.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from django_upgrade.data import Settings
+from tests.fixers.tools import check_noop, check_transformed
+
+settings = Settings(target_version=(4, 0))
+
+
+def test_not_settings_file():
+    check_noop(
+        """\
+        USE_L10N = True
+        """,
+        settings,
+    )
+
+
+def test_false():
+    check_noop(
+        """\
+        USE_L10N = False
+        """,
+        settings,
+        filename="myapp/settings.py",
+    )
+
+
+def test_dynamic():
+    check_noop(
+        """\
+        import os
+        USE_L10N = os.environ["USE_L10N"]
+        """,
+        settings,
+        filename="myapp/settings.py",
+    )
+
+
+def test_ignore_conditional():
+    check_noop(
+        """\
+        if something:
+            USE_L10N = True
+        """,
+        settings,
+        filename="myapp/settings.py",
+    )
```

Comment:
Added this case and the appropriate conditional `isinstance(parent, ast.Module)`, because erasing the line when it's the only thing in a block can leave the file broken

Indented settings can occur with e.g. django-configurations, I don't wanna worry about handling such situations (right now)